src/sagemaker_inference/etc/log4j2.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{ISO8601} [%-5p] %t %c - %m%n"/>
+        </Console>
+        <RollingFile
+                name="access_log"
+                fileName="${env:LOG_LOCATION:-logs}/access_log.log"
```

Comment:
Is it environment variables or system properties that were referenced in log4j.properties before? Do you know where LOG_LOCATION is set?

Reply:
The LOG_LOCATION is hardcoded. We use the file [here](https://github.com/aws/sagemaker-inference-toolkit/tree/master/src/sagemaker_inference/etc) to copy to the appropriate location in the container. It is not picked up using environment variables or system properties. Env variable would require a bit more work, testing and doc update.

Reply:
LOG_LOCATION used to be set in ConfigManager, but was removed as part of https://github.com/awslabs/multi-model-server/pull/979/files.

```
filePattern="${env:LOG_LOCATION:-logs}/mms_log.%d{dd-MMM}.log.gz">
```

this means, use `LOG_LOCATION` envar if it's set, otherwise use `logs` folder.

Clarified with @frankfliu.